kview/src/main/java/dev/ikm/komet/kview/mvvm/view/pattern/PatternDefinitionController.java

```diff
@@ -112,7 +121,9 @@ public class PatternDefinitionController {
     private VBox semanticOuterVBox;
 
     @FXML
-    private void clearView(ActionEvent actionEvent) {
+    private void clearView() {
+        patternDefinitionViewModel.setPropertyValue(PatternViewModel.PURPOSE_ENTITY, null);
+        patternDefinitionViewModel.setPropertyValue(PatternViewModel.MEANING_ENTITY, null);
     }
```

Comment:
Can we call `patternDefinitionViewModel.save()` or `.save(true)`? 
SimpleViewModels have a .save() and validation viewmodels have .save(boolean).

Reply:
updated